.pre-commit-config.yaml

```
@@ -0,0 +1,52 @@
+---
+repos:
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: "v0.0.275"
+    hooks:
+      - id: ruff
```

Add the name to make it more readable and beautiful when running
```suggestion
      - id: ruff
        name: Ruff (Python linter)
```